polaris-react/src/components/Badge/Badge.scss

```diff
@@ -11,29 +13,57 @@
   border-radius: var(--p-border-radius-5);
   color: var(--p-color-text);
 
+  :where(#{$se23}) & {
+    // stylelint-disable-next-line -- se23 bg-transparent-hover
+    background-color: var(--p-color-experimental-bg-transparent-subdued);
+    color: var(--p-color-text-subdued);
+    // stylelint-disable-next-line -- se23 border radius 10px
+    border-radius: 10px;
```

Comment:
In figma I don't see a difference between the `10px` radius and the `20px` old radius so I think you can probably keep the old token and delete this custom override! I don't think this change was intentional on the designers' end but you can ask to make sure